src/styles/blocks/card.scss

```diff
@@ -0,0 +1,94 @@
+.product-card {
+  box-sizing: border-box;
+  width: $card-width;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  color: $main-accent;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 16px;
+  transition: $transition-duration;
+
+  &__image {
+    display: block;
+    margin: auto;
+    margin-top: 16px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-top: 40px;
+  }
+
+  &__product-code {
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary;
+    margin-top: 4px;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: end;
+    margin-top: 16px;
+  }
+
+  &__reviews-count {
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__price-block {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 24px;
+  }
+
+  &__price-text {
+    margin: 0;
+    color: $secondary;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__price {
+    margin: 0;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+  }
+
+  &__button {
+    margin-top: 16px;
+    height: 40px;
+    border: none;
+    border-radius: 5px;
+    background-color: $blue-accent;
+    color: $white;
+    text-transform: uppercase;
+    font-size: 14px;
+    line-height: 16px;
+    font-weight: 700;
+    transition: $transition-duration;
```

Comment:
The transition property in &__button should include the CSS properties that the transition effect should apply to, such as 'background-color' and 'color'.